modules/monitoring/files/grafana-webhook.py

```diff
@@ -51,7 +51,7 @@ def post():
             finally:
                 lock.release()
                 lock_acquired = True
-    return (message, 200, None)
+    return '', 204
```

Comment:
Thought: could we maybe vary based on if it actually got the end of the try clause? 204 if success and 4/500? if failed?

Reply:
Done.